src/data/faq_de.json

```
@@ -376,6 +376,217 @@
+                                "anchor": "warn_without_tan_version",
+                                "active": false,
+                                "textblock": [
+                                    "Wenn Sie die Corona-Warn-App nicht auf das neue Release aktualisieren, wird für eine Übergangsfrist (bis zum ??????h) noch die Verifikations-Hotline zur Verfügung stehen, bei der Sie sich eine TAN ausstellen lassen können.",
+                                    "Nach diesem Zeitpunkt werden entweder die Testergebnisse nur noch direkt digital von den Teststellen in die CWA übertragen, wie bisher haben Sie dann die Möglichkeit, andere zu warnen. Oder Sie erhalten das Testergebnis auf anderem Wege. Dann können Sie – nach einem Update auf Release 3.0 – manuell mit der neuen Funktion \"Tests erfassen und andere warnen\" andere direkt warnen.",
+                                    "Für alle anderen Funktionen unterscheidet sich das neue Release von der vorherigen Version der CWA nicht.",
+                                    "Siehe auch: <a href='placeholder' target='_blank'>Wie kann ich andere Nutzer warnen, wenn ich positiv getestet worden bin?</a>"
+                                ]
+                            },
+                            {
+                                "title": "Muss ich einen PCR-Test machen, nachdem ich mit einem Selbsttest (positives Ergebnis) gewarnt habe?",
+                                "anchor": "warn_without_tan_pcr_after_self_test",
+                                "active": false,
+                                "textblock": [
+                                    "Aus epidemiologischer Sicht ist ein PCR-Test natürlich deutlich genauer und aussagekräftiger als ein von Laien durchgeführter Selbsttest. Daher wird zur Bestätigung (bzw. zur Überprüfung von ggf. False-Positives) des Selbsttests ein PCR-Test empfohlen.",
+                                    "Um die lokalen Bestimmungen zu befolgen, empfehlen wir daher, dass sie nach einem positiven Selbsttest Kontakt mit dem zuständigen Gesundheitsamt oder dem Arzt Ihres Vertrauens aufnehmen und das weitere Vorgehen besprechen."
+                                ]
+                            },
+                            {
+                                "title": "Was passiert, wenn ich nach einem positiven Selbsttest noch einmal mit einem positiven PCR-Test warne? Geht das überhaupt?",
+                                "anchor": "warn_without_tan_warn_twice",
+                                "active": false,
+                                "textblock": [
+                                    "Die neue Funktion lässt nach einem erfassten Selbsttest nur noch einen weiteren registrierten PCR-/Schnelltest-Test zu. Falls mit dem positiven Selbsttest schon gewarnt wurde, würden durch einen solchen weiteren „offiziellen“ Test mit positivem Ergebnis dann ggf. einige der Kontakte eine weitere Risikowarnung erhalten.",

```

1. Improved wording
2. Technical question: If, let's say, I test myself positive on 11.01.23 and warn others that day using the SRS feature. On 12.01.23 I take a PCR test, register it in the app and on 13.01.23 I get my positive result in the app. If I now warn others again, will only the contacts I had between I warned on 11.01.23 and the warning on 13.01.23 get a new exposure warning, or will the contacts I already warned on 11.01.23 also receive another exposure warning? If the last one is true, how will this be displayed in their apps (both in the contact diary & on the "Status" tab)? 
cc @thomasaugsten for this


```suggestion
                                    "Die neue Funktion lässt nach einem erfassten Selbsttest nur noch einen weiteren registrierten PCR-Test oder Schnelltest zu. Falls mit dem positiven Selbsttest schon gewarnt wurde, würden durch einen solchen weiteren „offiziellen“ Test mit positivem Ergebnis dann ggf. einige der Kontakte eine weitere Risikowarnung erhalten.",
```

---

Your question will be forwarded, please stand by. Your suggestion will be implemented

---

We got feedback from developers. Only the contacts that were met after the last test has been submitted will be warned.